docs/docs/index.md

```diff
@@ -134,6 +134,35 @@ with q.connection.pipeline() as pipe:
 
 `Queue.prepare_data` accepts all arguments that `Queue.parse_args` does.
 
+### Batch Enqueueing
+
+When using `enqueue_many`, the `batch` argument can be used to return a batch object instead of a list of jobs:
+
+```python
+batch = q.enqueue_many(
+  [
+    Queue.prepare_data(count_words_at_url, ('http://nvie.com',), job_id='my_job_id'),
+    Queue.prepare_data(count_words_at_url, ('http://nvie.com',), job_id='my_other_job_id'),
+  ],
+  batch=True,
+  batch_id='my_batch'
+)
```

Comment:
I'm not quite sure about this API. It doesn't feel intuitive to me.

This would also make `enqueue_many()` return a list of jobs or a batch depending on its kwarg, which may be confusing.

Reply:
I reverted the changes to `enqueue_many` and added a new `create` class method to Batch. Users can add jobs to a batch after they've been created or enqueued.